src/python/createrepo_c/__init__.py

```diff
@@ -145,12 +145,19 @@ def __init__(self, filename, mode=MODE_READ,
 
 # Repomd class
 
+
 class Repomd(_createrepo_c.Repomd):
     def __init__(self, path=None):
```

Comment:
The other option would be to expose the `warningcb` directly.  That felt a little awkward since it would mix API styles (it's not the way you would typically do that sort of thing in Python) but I wouldn't have a problem with doing it that way if it's what you would prefer.

Reply:
I think this is fine, thanks!